sql/core/src/test/scala/org/apache/spark/sql/CsvFunctionsSuite.scala

```diff
@@ -577,4 +578,11 @@ class CsvFunctionsSuite extends QueryTest with SharedSparkSession {
       $"csv", schema_of_csv("1,2\n2"), Map.empty[String, String].asJava))
     checkAnswer(actual, Row(Row(1, "2\n2")))
   }
+
+  test("StructsToCsv should not generate codes beyond 64KB") {
```

Comment:
Could you clarify this test title, please. I don't see anything related to 64KB in the test.

Doesn't the test just duplicates existing one: `to_csv - struct`?

Reply:
Sure, `checkEvaluation` will execute this test in both modes without codegen and with codegen modes. In case of codegen mode, if the generated java code (method) will be larger than 64 kb in size it won't be able to compile it, and the test will fail, as Java has a 64kb limit on the size of methods.
It doesn't duplicate `to_csv - struct` test case, because the purpose of this test - is to generate big StructType(with 5000 literals in current case) and test it on `StructsToCsv` expression to be sure that generated java code doesn't exceed the limit of 64 kb in size.